package.json

```diff
@@ -12,7 +12,10 @@
     "test:ci": "yarn test:lint && yarn test:unit:ci",
     "test:lint": "eslint --ext .js,jsx,json ./scripts && stylelint 'scripts/**/*.js'",
     "test:unit": "yarn build && jest --verbose --coverage",
-    "test:unit:ci": "jest --verbose --coverage"
+    "test:unit:ci": "jest --verbose --coverage",
+    "build": "yarn build:cjs && yarn build:esm",
+    "build:cjs": "yarn exec babel src -d dist --ignore '**/*.test.js,**/*.test.jsx,**/*.stories.js,**/*.stories.jsx' --root-mode upward",
+    "build:esm": "BABEL_TYPE=esm yarn exec babel src -d esm --ignore '**/*.test.js,**/*.test.jsx,**/*.stories.js,**/*.stories.jsx' --root-mode upward"
```

Comment:
has this been taken from psammead?

Reply:
err. Maybe.

Reply:
anything odd in them? (I took at the many package things)